### ISSUE TITLE
question 

### ISSUE THREAD

**Issue by hzzzt** [aysegulcayir/matlab_studies_RLWeq#1]
What equation does the quintic b-spline program correspond to?

**Comment by aysegulcayir** [comment]
It was for numerical solution of the RLW equation.

**Comment by hzzzt** [comment]
![image](https://user-images.githubusercontent.com/90895094/143978278-651a35d2-ef80-4a04-b902-7f39b04bc115.png)
I just touched this field, sorry.   this ？

**Comment by hzzzt** [comment]
![image](https://user-images.githubusercontent.com/90895094/143979599-4c3821f3-f4eb-4730-bb4e-1e574699873d.png)
or THIS?

**Comment by aysegulcayir** [comment]
It was too long ago. I could not remember. But you can understand the
initial condition and definition of the equation in matlab codes.  I can
use a lot of Rlw eq.

**Comment by hzzzt** [comment]
Yes, I am now trying to understand these codes, thank you for your reply. Can I continue to ask you if I have any questions in the future?

**Comment by aysegulcayir** [comment]
Of course you can ask me. but I have not looked at them for almost 3
years, so remembering them is a little difficult. Which article or equation
do you work? In my codes I had defined after the constants, first the
equation, initial and boundary conditions, respectively.

**Comment by hzzzt** [comment]
Thank you very much for your reply. I followed you on Twitter (I don’t know if it’s you, but the name is the same) and I left a message to you. I’m a student and I have only recently started learning about numerical solutions of partial differential equations. I just started learning about me. I often have some questions but I don't know where to go to solve them.

**Comment by aysegulcayir** [comment]
I am not on Twitter 😊. you can follow me on Linkedin. Ok, I suggest some
articles for you. They can give you an idea.
https://onlinelibrary.wiley.com/doi/abs/10.1002/num.20540
https://onlinelibrary.wiley.com/doi/abs/10.1002/cnm.1036
https://onlinelibrary.wiley.com/doi/abs/10.1002/num.20285
https://onlinelibrary.wiley.com/doi/abs/10.1002/num.20201

**Comment by hzzzt** [comment]
Thank you very much, what Linkedin is, I am a Chinese student, but I can download it and follow you

**Comment by hzzzt** [comment]
https://www.linkedin.com/in/志涛-胡-608577227/  this？
